h/presenters.py

```diff
@@ -63,11 +63,34 @@ def __init__(self, annotation):
         else:
             self.document = None
 
+    def _get_selection(self):
+        selectors = self.annotation.target_selectors
+        for selector in selectors:
+            if "exact" in selector:
+                return selector["exact"]
+
     @property
     def uri(self):
         return jinja2.escape(self.annotation.target_uri)
 
     @property
+    def text_rendered(self):
+        """The body text of this annotation.
+
+        """
+
+        return self.annotation.text_rendered
+
+    @property
+    def quote(self):
```

Comment:
In the front-end code at least, we require functions to be consistent about returning a value or not returning a value. Other functions in this file return an empty string if the value is not present, I would suggest doing the same here.